queue/src/fiskaltrust.Middleware.Queue/Bootstrapper/LocalizedQueueBootStrapperFactory.cs

```diff
@@ -27,7 +28,7 @@ public static ILocalizedQueueBootstrapper GetBootstrapperForLocalizedQueue(Guid
                 "FR" => middlewareConfiguration.PreviewFeatures.TryGetValue("queue-fr", out var val) && val ? new QueueFRBootstrapper() : throw new NotImplementedException("The French Queue is not yet implemented in this version."),
                 "IT" => new QueueITBootstrapper(),
                 "ME" => new QueueMeBootstrapper(),
-                _ => throw new ArgumentException($"Unkown country code: {countyCode}"),
+                _ => new QueueDEFAULTBootstrapper(),
```

Comment:
IMO we should be careful here in case of future upgrades from the Queue.

e.g. we implement Croatia (countrycode HR) and someone is already using croatia with the generic queue. In this case a package upgrade that includes a working version for HR would cause a breaking change. So maybe we should only allow this "Default" implementation when explicitly defined?

Reply:
Agree 👍🏻 and can we please limit this to sandbox only? Production Queues should definitely not be allowed to use this in any case.

Reply:
In the last commits I've added your suggestions